app/src/components/fields/DateTimeFields.tsx

```diff
@@ -75,8 +91,8 @@ export const DateTimeFields = <FormikPropsType,>(props: IDateTimeFieldsProps<For
                 name: dateName,
                 required: dateRequired,
                 variant: 'outlined',
-                error: get(touched, dateName) && Boolean(get(errors, dateName)),
-                helperText: get(touched, dateName) && get(errors, dateName),
+                error: get(touched, dateName) && (dateError || Boolean(get(errors, dateName))),
+                helperText: get(touched, dateName) && (dateHelperText || get(errors, dateName)),
```

Comment:
Should we flip 'dateHelperText' and 'get(errors, dateName)' so the error will render if we are providing the dateHelperText?

```js
helperText: get(touched, dateName) && (get(errors, dateName) || dateHelperText)
```

Reply:
I think dateHelperText needs to be first. If there is an error, we want to render dateHelperText if it exists, and fall back to the normal error string if its not provided.